websites/F/Free Web Novel/presence.ts

```diff
@@ -0,0 +1,187 @@
+const presence = new Presence({
+		clientId: "965294297048023050"
+	}),
+	browsingTimestamp = Math.floor(Date.now());
+
+presence.on("UpdateData", async () => {
+	let [showCover, showButtons, showBook, showLogo] = await Promise.all([
+		presence.getSetting<boolean>("showCover"),
+		presence.getSetting<boolean>("showButtons"),
+		presence.getSetting<boolean>("showBook"),
+		presence.getSetting<boolean>("showLogo")
+	]);
+	const [privacy, showTimestamp, showReading] = await Promise.all([
+			presence.getSetting<boolean>("privacy"),
+			presence.getSetting<boolean>("showTimestamp"),
+			presence.getSetting<boolean>("showReading")
+		]),
+		presenceData: PresenceData = {
+			largeImageKey: "nocover",
+			smallImageKey: showReading ? "closed" : null,
+			smallImageText: showReading ? "Not Reading" : null
+		},
+		{ pathname } = window.location,
+		pathnames = [
+			"/",
+			"/history/",
+			"/latest-release-novel/",
+			"/latest-novel/",
+			"/completed-novel/",
+			"/most-popular-novel/",
+			"/search/",
+			"/contact.html",
+			"/sitemap.xml",
+			"/privacy-terms-of-use.html",
+			"/genre/",
+			"/author/"
+		];
+	if (showTimestamp) presenceData.startTimestamp = browsingTimestamp;
```

Comment:
If you are using a ternary in the presenceData object for the other values, you might as well include this too